### PR TITLE
Fix movement.xml and add disguised option to chams

### PR DESCRIPTION
**Pull request by LightPower392**
The capitalization of movement.xml and autoheal.xml have been fixed. Also, there is an option for ignore disguised in chams

### CODE REVIEW

File: src/visual/EffectChams.cpp [hunk 1]
```diff
@@ -24,7 +25,7 @@ static settings::Boolean stickies{ "chams.show.stickies", "true" };
 static settings::Boolean teammate_buildings{ "chams.show.teammate-buildings", "false" };
 static settings::Boolean recursive{ "chams.recursive", "true" };
 static settings::Boolean weapons_white{ "chams.white-weapons", "true" };
-static settings::Boolean legit{ "chams.legit", "false" };
+static settings::Boolean legit{ "chams.legit", "true" };
```
Comment (TotallyNotElite) [inline src/visual/EffectChams.cpp]: Please avoid changing the defaults of variables without justification.

File: src/visual/EffectChams.cpp [hunk 2]
```diff
@@ -233,6 +234,8 @@ bool EffectChams::ShouldRenderChams(IClientEntity *entity)
     case ENTITY_PLAYER:
         if (!players)
             return false;
+        if (!disguised && (IsPlayerDisguised(ent)))
```
Comment (TotallyNotElite) [inline src/visual/EffectChams.cpp]: Should be expressed as `if (!disguised && IsPlayerDisguised(ent))` instead. The extra brackets are unnecessary.

**Comment by LightPower392** [comment]
Removed the extra brackets from both chams and glow. Also reverted the default legit chams option.

**Comment by TotallyNotElite** [comment]
Thanks for contributing! Your changes are currently on the master branch but not yet available to stable branch users. They will be included in the next stable update.

**Pull request merged**